Nodejs/Product/TestAdapter/TestExecutorWorker.cs

```diff
@@ -316,11 +318,47 @@ void RecordEnd(TestCaseResult test, ResultObject resultObject)
             }
         }
 
+        private int GetParentProcessId(uint processId, int counter = 0)
+        {
+            var query = string.Format("SELECT ParentProcessId FROM Win32_Process WHERE ProcessId = {0}", processId);
+            var search = new ManagementObjectSearcher("root\\CIMV2", query);
+            var results = search.Get().GetEnumerator();
+            results.MoveNext();
+
+            var parentId = (uint)results.Current["ParentProcessId"];
```

Comment:
Could the cast throw an exception if there's no `ParentProcessId` or will always have a parent?

Reply:
There will always be a parent, but I'm still making the try block include everything (commit coming) because that makes more sense.